mikeio1d/pandas_extension/mikeio1d_accessor.py

```diff
@@ -0,0 +1,70 @@
+import pandas as pd
+
+from .transposed_groupby import TransposedGroupBy
+from .result_reaches_helpers import agg_chainage
+from .result_reaches_helpers import groupby_chainage
+from .various import compact_dataframe
+
+
+@pd.api.extensions.register_dataframe_accessor("m1d")
+class Mikeio1dAccessor:
```

Comment:
Add a docstring potentially with a link where it is possible to read more about this API?